### PR TITLE
Authentication + account pages

### PR DESCRIPTION
**Pull request by elsmr**
send help

### CODE REVIEW

**Comment by Haroenv** [comment]
Looking pretty awesome already ❤️

File: src/main/resources/js/main.js [hunk 1]
```diff
@@ -1,7 +1,28 @@
-/* global options, d3pie */
+const logout = document.getElementById('logout');
 
-var pie = new d3pie('results', { // eslint-disable-line new-cap
-	data: {
-		content: options
-	}
+logout.addEventListener('click', (e) => {
+	httpPost('/logout')
```
Comment (Haroenv) [inline src/main/resources/js/main.js]: stomme bot

File: src/test/java/com/pollistics/services/UserServiceTests.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+@SpringBootTest(classes = PollisticsApplication.class)
+public class UserServiceTests {
+	@Autowired
+	private UserRepository userRepo;
+
+	@Autowired
+	private UserService userService;
+
+	@Test
+	public void createUserTest() {
```
Comment (Haroenv) [inline src/test/java/com/pollistics/services/UserServiceTests.java]: stomme codacy sterf

**Review by Maliek — APPROVED** [review]
Looks good my negev.

**Review by Haroenv — APPROVED** [review]
A P P R O V E 

A F

**Pull request merged**